blueprints/controllers/ikea_e1744/ikea_e1744.yaml

```
@@ -215,11 +241,9 @@ action:
           # repeat until the automation is restarted ie. when the corresponding stop message is received
           - repeat:
               sequence: !input action_rotate_left
-              until: '{{ not rotate_left_loop }}'
+              while: '{{ rotate_left_loop }}'
       - conditions:
           - '{{ trigger_action in rotate_left_stop }}'
-          # for integrations which need to store the last controller event, need to check the previous button event, stored in the provided input_text
-          - '{{ not integration_id in integrations_with_prev_event_storage or states(helper_last_controller_event) in rotate_left }}'
         sequence:
```

```suggestion
          # for integrations which need to store the last controller event, need to check the previous button event, stored in the provided input_text
          - '{{ not integration_id in integrations_with_prev_event_storage or last_controller_event in rotate_left }}'
        sequence:
```

Don't have a way to properly test this, but it should work to identify which stop action to execute.

---

I removed this because we need to make sure a condition is always matched, because otherwise it could happen a ahb_controller stop message is not send out causing continues loop.

---

The issue is that if we receive a stop action, we'll always execute the `rotate_left_stop` action for ZHA or Zigbee2MQTT due to the order of choose steps. This is an erroneous behaviour, since there would not be a way to execute the `rotate_right_stop` action.